package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/dtdl-parser",
-  "version": "0.2.29",
```

Comment:
I think it should be a major bump since it's changing the return type of an export